dangerzone/gui/main_window.py

```diff
@@ -486,7 +486,7 @@ def start_button_clicked(self):
 
 
 class ConvertThread(QtCore.QThread):
-    finished = QtCore.Signal(bool)
+    is_finished = QtCore.Signal(bool)
```

Comment:
Per my testing the signal `finished` is emitted anyways from a QThread when it finishes anyways. So we don't even need it. This avoids the whole typing issue.